src/skadi/sample.cc

```diff
@@ -334,7 +376,7 @@ tile_data cache_t::source(uint16_t index) {
   }
 
   // item in cache is already unpacked
-  const char* unpacked = item.get_unpacked();
```

Comment:
please revert this change, or use `const auto *`. auto is great and all but when we use it to hide qualifiers it makes everyones lives harder

Reply:
Whoops, that was an oversight. Good catch!